plugins/nomad/src/components/EntityNomadAllocationListTable/EntityNomadAllocationListTable.tsx

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright 2020 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  Link,
+  MissingAnnotationEmptyState,
+  ResponseErrorPanel,
+  StatusError,
+  StatusOK,
+  StatusPending,
+  StatusRunning,
+  Table,
+  TableColumn,
+} from '@backstage/core-components';
+import { useEntity } from '@backstage/plugin-catalog-react';
+import React, { useState } from 'react';
+import { Allocation, nomadApiRef } from '../../api';
+import { configApiRef, useApi } from '@backstage/core-plugin-api';
+import {
+  NOMAD_GROUP_ANNOTATION,
+  NOMAD_JOB_ID_ANNOTATION,
+  NOMAD_NAMESPACE_ANNOTATION,
+  isNomadAllocationsAvailable,
+} from '../../Router';
+import useAsync from 'react-use/lib/useAsync';
+
+type rowType = Allocation & { nomadAddr: string };
+
+const columns: TableColumn<rowType>[] = [
+  {
+    title: 'ID',
+    field: 'ID',
+    render: row => (
+      <Link to={`${row.nomadAddr}/ui/allocations/${row.ID}`} underline="always">
+        {row.ID.split('-')[0]}
+      </Link>
+    ),
+  },
+  {
+    title: 'Task Group',
+    field: 'TaskGroup',
+    render: row => (
+      <Link
+        to={`${row.nomadAddr}/ui/jobs/${row.JobID}/${row.TaskGroup}`}
+        underline="always"
+      >
+        {row.TaskGroup}
+      </Link>
+    ),
+  },
+  {
+    title: 'Created',
+    field: 'CreateTime',
+    render: row => new Date(row.CreateTime / 1000000).toLocaleString(),
```

Comment:
there's actually an ADR defining an opiated way Backstage and ideally  it's plugins should display these?

may be good to align it!

https://backstage.io/docs/architecture-decisions/adrs-adr012

Reply:
Ah noice, thanks for sharing. I've updated the `DateTime` parsing to use `luxon`!

<img width="207" alt="Screenshot 2023-06-09 at 10 54 38 PM" src="https://github.com/backstage/backstage/assets/13923102/fe73ee02-5f9f-491c-a0d4-8d4367d005dd">